apps/nuxt3-ssr/package.json

```diff
@@ -14,7 +14,8 @@
     "preview": "nuxt preview",
     "postinstall": "if-env CI=true && nuxt prepare || echo 'CI is not set'",
     "format": "prettier components composables gql interfaces pages plugins utils tests  --write --config ../.prettierrc.js --gitignore",
-    "checkFormat": "prettier components composables gql interfaces pages plugins utils tests  --check --config ../.prettierrc.js --gitignore"
+    "checkFormat": "prettier components composables gql interfaces pages plugins utils tests  --check --config ../.prettierrc.js --gitignore",
+    "typecheck": "nuxi typecheck"
```

Comment:
Maybe make it checkType to be in line with the checkFormat command. Otherwise typeCheck since 2 words.